doc/howto/config.rst

```diff
@@ -502,6 +502,45 @@ Example::
     }
 
 
+requested_attributes
+""""""""""""""""""""
+
```

Comment:
This is relevant only for the eIDAS SAML profile. 
A list of attributes that the SP requests from an eIDAS-Service (IdP). These attributes are included in the SAML Authentication Request that this SP sends as it is shown in the example below : 
``` xml
<?xml version="1.0" encoding="UTF-8"?>
<saml2p:AuthnRequest
 Destination="https://eidas-service.eu/post"
 ID="_171ccc6b39b1e8f6e762c2e4ee4ded3a" IssueInstant="2015-04-30T19:25:14.273Z" Version="2.0"
 xmlns:saml2p="urn:oasis:names:tc:SAML:2.0:protocol" xmlns:eidas="http://eidas.europa.eu/samlextensions">
.........
<saml2p:Extensions>
 <eidas:SPType>public</eidas:SPType>
 <eidas:RequestedAttributes>
 <eidas:RequestedAttribute Name="http://eidas.europa.eu/attributes/naturalperson/PersonIdentifier"
NameFormat="urn:oasis:names:tc:SAML:2.0:attrname-format:uri" isRequired="true"/>
 <eidas:RequestedAttribute Name="http://eidas.europa.eu/attributes/naturalperson/CurrentFamilyName"
NameFormat="urn:oasis:names:tc:SAML:2.0:attrname-format:uri" isRequired="true"/>
 <eidas:RequestedAttribute Name="http://eidas.europa.eu/attributes/naturalperson/CurrentGivenName"
NameFormat="urn:oasis:names:tc:SAML:2.0:attrname-format:uri" isRequired="true"/>
 <eidas:RequestedAttribute Name="http://eidas.europa.eu/attributes/naturalperson/DateOfBirth"
NameFormat="urn:oasis:names:tc:SAML:2.0:attrname-format:uri" isRequired="true"/>
 </eidas:RequestedAttributes>
 </saml2p:Extensions>
.........
</saml2p:AuthnRequest>
```

Please note that this is different from the `required_attributes` and `optional_attributes` parameters that control the requested attributes in the metadata of an SP.